src/Movies/MoviesList.tsx

```diff
@@ -0,0 +1,25 @@
+import { MovieModel } from './movie.model.ts';
+import styles from './MoviesList.module.css';
+import { ReactElement } from 'react';
+import { MovieTile } from './MovieTile.tsx';
+import { ContextMenu } from '../components';
+
+interface MoviesListProps {
+	movies: MovieModel[];
+	onMovieClick: (movie: MovieModel) => void
+}
+
+export function MoviesList({ movies, onMovieClick }: MoviesListProps): ReactElement {
+	const contextMenuItems = [
```

Comment:
If you have no dependencies on component its better to move this array outside the component or in separate file to not create it on each render